pkg/minikube/out/register/log.go

```diff
@@ -154,3 +155,26 @@ func NewErrorExitCode(err string, exitcode int, additionalData ...map[string]str
 func (s *Error) Type() string {
 	return "io.k8s.sigs.minikube.error"
 }
+
+// Audit represents the execution of a command
+type Audit struct {
```

Comment:
we can move these to audit package and it be called something like

Entry

so it would ready as
audit.Entry if it is referenced outside ( most porbaby would not need to be public though)

Reply:
Good idea, done!